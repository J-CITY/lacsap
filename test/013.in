{$ qwer}
program HelloWorld;{
 $QWER
}
{
   $ QWERTY

}
var
    _int, lazy_town_ : integer;
    __r : real;
Begin

  int := 1;
END.
{$
q
b
c
d
e
