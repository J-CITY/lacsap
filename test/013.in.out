            'ajsq__'   0   0     literalChar    27
